docs/389ds/design/accesslog-json-design.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Access Log JSON Format Design"
+---
+
+# Access Log JSON Format
+----------------
+
+Overview
+--------
+
+In order to make the access logging more consumable by standard parsing tools there should be an option for writing the log in JSON
+
+JSON Design
+------------
+
+    {
+        // Header
+        version: "",
```

Comment:
What is the `version` field? Is it the 389 DS version?
If so, should we also have something like `format_version`, which could make future format changes easier to manage?

Reply:
> What is the `version` field? Is it the 389 DS version? If so, should we also have something like `format_version`, which could make future format changes easier to manage?

This was to address Thierry's point about the header that the current logs have:

        389-Directory/3.1.1 DEVELOPER BUILD B0000.000.0000
        localhost.localdomain:636 (/etc/dirsrv/slapd-localhost)